kubejs/data/occultism/loot_tables/entities/wild_hunt_wither_skeleton.json

```diff
@@ -0,0 +1,263 @@
+                            "condition": "killed_by_player"
+                        }
+                    ],
+                    "functions": [
+                        {
+                            "function": "enchant_with_levels",
+                            "levels": 50,
+                            "treasure": true
+                        }
+                    ]
+                },
+                {
+                    "type": "item",
+                    "weight": 1,
+                    "name": "apotheosis:armor_feet_book",
+                    "conditions": [
+                        {
+                            "condition": "killed_by_player"
+                        }
+                    ],
+                    "functions": [
+                        {
+                            "function": "enchant_with_levels",
+                            "levels": 50,
+                            "treasure": true
+                        }
+                    ]
+                },
+                {
+                    "type": "item",
+                    "weight": 1,
+                    "name": "apotheosis:digger_book",
+                    "conditions": [
+                        {
+                            "condition": "killed_by_player"
+                        }
+                    ],
+                    "functions": [
+                        {
+                            "function": "enchant_with_levels",
+                            "levels": 50,
+                            "treasure": true
+                        }
+                    ]
+                },
+                {
+                    "type": "item",
+                    "weight": 1,
+                    "name": "apotheosis:fishing_rod_book",
+                    "conditions": [
+                        {
+                            "condition": "killed_by_player"
+                        }
+                    ],
+                    "functions": [
+                        {
+                            "function": "enchant_with_levels",
+                            "levels": 50,
+                            "treasure": true
+                        }
+                    ]
+                },
+                {
+                    "type": "item",
+                    "weight": 1,
+                    "name": "apotheosis:bow_book",
+                    "conditions": [
+                        {
+                            "condition": "killed_by_player"
+                        }
+                    ],
+                    "functions": [
+                        {
+                            "function": "enchant_with_levels",
+                            "levels": 50,
+                            "treasure": true
+                        }
+                    ]
+                },
+                {
+                    "type": "loot_table",
+                    "weight": 20,
+                    "name": "minecraft:chests/nether_bridge",

```

Comment:
Something to keep in mind is that Artifacts actually adds some items to this loot table, so the chance of getting an artifact will be higher than 1/49, and it will be biased in favor of the items artifacts adds to this loot table (see [artifacts:loot_tables/inject/chest/nether_bridge.json](https://github.com/ochotonida/artifacts/blob/1.16/src/generated/resources/data/artifacts/loot_tables/inject/chests/nether_bridge.json))